Earthfile

```diff
@@ -55,7 +55,13 @@ devcontainer-base:
             cmake \
             ninja-build \
             pkg-config \
-            libgtk2.0-dev \
+            libgtk-3-dev \
+            libgc-dev \
+            libpcre3-dev \
+            zlib1g-dev \
+            apache2-dev \
+            libmysqlclient-dev \
+            libsqlite3-dev \
             $LINK_DYNAMIC_PACKAGES \
```

Comment:
Please only add `libgtk-3-dev` here, the rest are set in the `$LINK_DYNAMIC_PACKAGES` variable.

Reply:
OK :+1: :slightly_smiling_face: